src/Compilers/CSharp/Test/Semantic/Semantics/SpanStackSafetyTests.cs

```diff
@@ -264,9 +264,9 @@ void M<T>(T t)
     }
 }");
             comp.VerifyDiagnostics(
-                // (7,13): error CS0019: Operator 'is' cannot be applied to operands of type 'T' and 'Span<int>'
+                // (7,13): warning CS0184: The given expression is never of the provided ('Span<int>') type
                 //         if (t is Span<int>)
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "t is Span<int>").WithArguments("is", "T", "System.Span<int>").WithLocation(7, 13),
+                Diagnostic(ErrorCode.WRN_IsAlwaysFalse, "t is Span<int>").WithArguments("System.Span<int>").WithLocation(7, 13),
```

Comment:
>WRN_IsAlwaysFalse [](http://example.com/codeflow?start=37&length=17)

The severity of the diagnostic for `t is Span<int>` was reduced from an error to a warning. Will that affect success cases in C#12 (say overload resolution with lambda arguments)?

Reply:
> > WRN_IsAlwaysFalse
> 
> The severity of the diagnostic for `t is Span<int>` was reduced from an error to a warning. Will that affect success cases in C# 12 (say overload resolution with lambda arguments)?

Quite possibly, but we, usually, do not worry about changes in behavior for scenarios like that.